### PR TITLE
GUVNOR-3318 : [GSS](6.4.x)Incorrect Warning Messages for Conflicting Rows in Guided DT

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/GUVNOR-3318

-Single hit and Conflict checks ignored BRL conditions

### CODE REVIEW

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/cache/inspectors/RuleInspector.java [hunk 1]
```diff
@@ -225,28 +223,29 @@ public RuleInspectorCache getCache() {
...
     @Override
-    public boolean conflicts( final Object other ) {
-        if ( other instanceof RuleInspector ) {
-            if ( getActionsInspectors().conflicts( ( (RuleInspector) other ).getActionsInspectors() ) ) {
-                if ( getConditionsInspectors().subsumes( ( (RuleInspector) other ).getConditionsInspectors() ) ) {
+    public boolean conflicts(final Object other) {
+        if (other instanceof RuleInspector) {
+            if (getActionsInspectors().conflicts(((RuleInspector) other).getActionsInspectors())) {
+                if (getConditionsInspectors().subsumes(((RuleInspector) other).getConditionsInspectors())
+                        && getBrlConditionsInspectors().subsumes(((RuleInspector) other).getBrlConditionsInspectors())) {
```
Comment (Rikkola) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/cache/inspectors/RuleInspector.java]: Fix is here. Rest is formatting.

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/SingleHitCheck.java [hunk 2]
```diff
@@ -41,7 +41,10 @@ protected CheckType getCheckType() {
 
     @Override
     public boolean check() {
-        return hasIssues = ruleInspector.getConditionsInspectors().subsumes( other.getConditionsInspectors() );
+        return hasIssues =
+                ruleInspector.getConditionsInspectors().subsumes(other.getConditionsInspectors())
+                        && ruleInspector.getBrlConditionsInspectors().subsumes(other.getBrlConditionsInspectors
+                        ());
```
Comment (Rikkola) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/SingleHitCheck.java]: Other part of the fix.

**Comment by Rikkola** [comment]
@jhrcek Hi, you are marked as the tester for RHBPMS-4817. This is the PR for it. Not sure if you or @jomarko wants to take a look.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Pull request merged**